packages/back-end/types/stats.d.ts

```diff
@@ -60,8 +60,15 @@ export type ExperimentMetricAnalysis = {
   }[];
 }[];
 
+export type BanditResponse = {
+  banditUpdateMessage?: string;
+  banditWeights?: number[];
```

Comment:
try to return weights at all times (even if 50/50), but keep it optional for cases when stats engine can't reliably make an assumption